lms/services/oauth_http.py

```diff
@@ -0,0 +1,49 @@
+class OAuthHTTPService:
```

Comment:
Something about the name `OAuthHTTPService` is a little odd to me. It's not a service for "doing" OAuth, so much as doing something else which happens to use `OAuth`. I can't think of anything better that isn't long (like `OAuthAuthenticatedHTTPService`), I don't think it's going to cause too much confusion though.

Reply:
`OAuthRequestsService`a maybe. Or `OAuthClient`